kibana/saved-objects/kibana-8-x-alerts.ndjson

```diff
@@ -1,16 +1,15 @@
+{"excludedObjects":[],"excludedObjectsCount":0,"exportedCount":14,"missingRefCount":0,"missingReferences":[]}
```

Comment:
shall we add a newline

Reply:
Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/14